platforms/core-configuration/configuration-cache/src/test/kotlin/org/gradle/configurationcache/fingerprint/ConfigurationCacheFingerprintCheckerTest.kt

```diff
@@ -201,16 +266,34 @@ class ConfigurationCacheFingerprintCheckerTest {
         )
     }
 
+    data class FileInfo(
+        val file: File,
+        val hashCode: HashCode,
+        val fileType: FileType
+    )
+
     private
     fun invalidationReasonForInitScriptsChange(
         from: Iterable<Pair<File, HashCode>>,
         to: List<Pair<File, HashCode>>
-    ): InvalidationReason? = to.toMap().let { toMap ->
+    ): InvalidationReason? =
+        invalidationReasonForFileSystemChange(
+            from.map { FileInfo(it.first, it.second, FileType.RegularFile) },
+            to.map { FileInfo(it.first, it.second, FileType.RegularFile) })
```

Comment:
💅 Move closing paren to its own line for consistency with the surrounding code

Reply:
Thanks, I ended up reverting introduction to this helper method, as it was not really used in the new tests